### ISSUE TITLE
Use Vega and Vega-Lite from jsdelivr

### ISSUE THREAD

**Issue by domoritz** [vega/editor#336]

**Comment by algomaster99** [comment]
@domoritz When the jsdelivr links are used, how are they imported? One method can be to import it in `componentWillMount` but that will be highly redundant and inefficient.

**Comment by domoritz** [comment]
I think we should do this in webpack somehow or with lazy imports (maybe later).

**Comment by siddhant1** [comment]
Loading vega and vega-lite from cdn?

**Comment by domoritz** [comment]
Yes, in production only, though (or have a somehow the npm version as default so VL and VG devs can use the editor)!

**Comment by joelostblom** [comment]
This would be helpful when troubleshooting from altair as well. Now there can be a difference between the version used by altair to render the spec and the version used when clicking "show in vega editor", which can make it confusing to troubleshoot. Example here related to https://github.com/vega/vega-lite/issues/9078: 

https://github.com/vega/editor/assets/4560057/6e727719-64d8-4bb6-aa7a-ec411480fa9d

**Comment by domoritz** [comment]
Here is how we could do it maybe https://bsky.app/profile/rich-harris.dev/post/3llygloabis2h
